elliott/elliottlib/cli/verify_attached_bugs_cli.py

```diff
@@ -312,62 +375,100 @@ async def _verify_attached_flaws_for(self, advisory_id: int, attached_trackers:
+                    msg = (f"On advisory {advisory_id}, {cve} is associated with Brew components "
+                           f"{', '.join(sorted(cve_package_exclusions))} without a tracker bug."
+                           " You may need to explicitly exclude those Brew components from the CVE "
+                           "mapping or attach the corresponding tracker bugs.")
+                    issue = VerifyIssue(
+                        code=VerifyIssueCode.MISSING_CVE_EXCLUSIONS,
+                        message=msg,
+                        advisory=advisory_id,
+                    )
+                    issues.append(issue)
         except ValueError as e:
-            self._complain(e)
+            issues.append(VerifyIssue(
+                code=VerifyIssueCode.VALIDATION_ERROR,
+                message=str(e),
+                advisory=advisory_id,
+            ))
 
         # Validate `CVE Names` field of the advisory
         advisory_cves = advisory_info["content"]["content"]["cve"].split()
         extra_cves = cve_components_mapping.keys() - advisory_cves
         if extra_cves:
-            self._complain(f"On advisory {advisory_id}, bugs for the following CVEs are already attached "
-                           f"but they are not listed in advisory's `CVE Names` field: {', '.join(sorted(extra_cves))}")
+            msg = (f"On advisory {advisory_id}, bugs for the following CVEs are already attached "
+                   f"but they are not listed in advisory's `CVE Names` field: {', '.join(sorted(extra_cves))}")
+            issue = VerifyIssue(
+                code=VerifyIssueCode.EXTRA_CVE_NAMES_IN_ADVISORY,
+                message=msg,
+                advisory=advisory_id,
+            )
+            issues.append(issue)
         missing_cves = advisory_cves - cve_components_mapping.keys()
         if missing_cves:
-            self._complain(f"On advisory {advisory_id}, bugs for the following CVEs are not attached but listed in "
-                           f"advisory's `CVE Names` field: {', '.join(sorted(missing_cves))}")
+            msg = (f"On advisory {advisory_id}, bugs for the following CVEs are not attached but listed in "
+                   f"advisory's `CVE Names` field: {', '.join(sorted(missing_cves))}")
+            issue = VerifyIssue(
+                code=VerifyIssueCode.MISSING_CVE_NAMES_IN_ADVISORY,
+                message=msg,
+                advisory=advisory_id,
+            )
+            issues.append(issue)
+
+        self._complain(issues)
 
     def get_attached_bugs(self, advisory_ids: List[str]) -> Dict[int, Set[Bug]]:
         """ Get bugs attached to specified advisories
         :return: a dict with advisory id as key and set of bug objects as value
         """
         logger.info(f"Retrieving bugs for advisories: {advisory_ids}")
-        advisories = [Erratum(errata_id=advisory_id) for advisory_id in advisory_ids]
```

Comment:
This is very slow for large advisories as it tries to load all errata builds. So instead fetch raw erratum and bugs only.